app/app.js

```diff
@@ -68,10 +69,17 @@ async function init() {
       block.appendChild(image);
 
       if (attributes["WEBSITE"]) {
-        const websiteChip = document.createElement("calcite-chip");
-        websiteChip.id = "detail-chip-website";
-        websiteChip.innerText = `website: ${attributes["WEBSITE"]}`;
-        block.appendChild(websiteChip);
+        const itemWebsite = document.createElement("calcite-button");
+        itemWebsite.setAttribute("id", "detail-item-website");
```

Comment:
nitpick: I don't think we need to call setAttribute. We can just set the property.

Reply:
Updated.